package.json

```diff
@@ -15,6 +17,7 @@
   },
   "devDependencies": {
     "jest": "^23.5.0",
-    "nodemon": "^1.18.3"
+    "nodemon": "^1.18.3",
+    "prettier": "1.14.2"
```

Comment:
I moved prettier to `devDependencies` since it's not needed for production code.